avahi-gobject/ga-entry-group.h

```diff
@@ -32,7 +32,7 @@ typedef enum {
     GA_ENTRY_GROUP_STATE_UNCOMMITED = AVAHI_ENTRY_GROUP_UNCOMMITED,
     GA_ENTRY_GROUP_STATE_REGISTERING = AVAHI_ENTRY_GROUP_REGISTERING,
     GA_ENTRY_GROUP_STATE_ESTABLISHED = AVAHI_ENTRY_GROUP_ESTABLISHED,
-    GA_ENTRY_GROUP_STATE_COLLISTION = AVAHI_ENTRY_GROUP_COLLISION,
+    GA_ENTRY_GROUP_STATE_COLLISION = AVAHI_ENTRY_GROUP_COLLISION,
```

Comment:
Changing this variable name would potentially change the interface for the GObject binding to Avahi, I think. So it should not be part of this multi-file spelling correction effort - for this change, specifically any binding/side effects needs to be known and discussed prior to changing (potentially) the API here.